homeassistant/components/transmission/sensor.py

```diff
@@ -55,43 +60,50 @@ class TransmissionSensorEntityDescription(SensorEntityDescription):
 STATUS_SENSOR_DESCRIPTIONS = [
     TransmissionSensorEntityDescription(
         key="status",
-        name_suffix="Status",
+        name="Status",
     ),
 ]
 TORRENTS_SENSOR_DESCRIPTIONS = [
     TransmissionSensorEntityDescription(
         key="active_torrents",
-        name_suffix="Active Torrents",
+        name="Active Torrents",
         sub_type="active",
         state_class=SensorStateClass.MEASUREMENT,
     ),
     TransmissionSensorEntityDescription(
         key="paused_torrents",
-        name_suffix="Paused Torrents",
+        name="Paused Torrents",
         sub_type="paused",
         state_class=SensorStateClass.MEASUREMENT,
     ),
     TransmissionSensorEntityDescription(
         key="total_torrents",
-        name_suffix="Total Torrents",
+        name="Total Torrents",
         sub_type="total",
         state_class=SensorStateClass.MEASUREMENT,
     ),
     TransmissionSensorEntityDescription(
         key="completed_torrents",
-        name_suffix="Completed Torrents",
+        name="Completed Torrents",
         sub_type="completed",
         state_class=SensorStateClass.MEASUREMENT,
     ),
     TransmissionSensorEntityDescription(
         key="started_torrents",
-        name_suffix="Started Torrents",
+        name="Started Torrents",
         sub_type="started",
         state_class=SensorStateClass.MEASUREMENT,
     ),
 ]
 
 
+def get_unique_id(
+    config_entry: ConfigEntry, entity_description: TransmissionSensorEntityDescription
+) -> str:
+    """Generate a unique id for entity."""
+    return f"{config_entry.entry_id}-{entity_description.key}"
```

Comment:
We don't need to have a method for this.
Since this method will just directly return the string, just use that string in the places needed directly (aka this is an unneeded "proxy" method, that only adds to complexity).

Reply:
The idea was to guarantee consistency between `attr_unique_id` and migration code. But sure, I'll remove the helper function.